rust/crd/src/lib.rs

```diff
@@ -331,10 +339,16 @@ impl AirflowRole {
         ];
 
         match &self {
-            AirflowRole::Webserver => command.extend(vec![
-                "prepare_signal_handlers".to_string(),
-                "airflow webserver &".to_string(),
-            ]),
+            AirflowRole::Webserver => {
+                // Getting auth commands for AuthClass
+                let auth_commands = vec![Self::authentication_start_commands(auth_config)];
+                command.extend(vec![
+                    "prepare_signal_handlers".to_string(),
+                    "airflow webserver &".to_string(),
+                ]);
+                command.extend(auth_commands);
```

Comment:
The TLS certificates must be set before the webserver uses them. So, the according command should be placed before starting the webserver:

```suggestion
                command.extend(Self::authentication_start_commands(auth_config));
                command.extend(vec![
                    "prepare_signal_handlers".to_string(),
                    "airflow webserver &".to_string(),
                ]);
```

Reply:
done https://github.com/stackabletech/airflow-operator/pull/530/commits/b4d0e2fb818c5144a293d0583d4232eef9216661